doc/source/user_guide/computation.rst

```diff
@@ -361,6 +361,9 @@ compute the mean absolute deviation on a rolling basis:
    @savefig rolling_apply_ex.png
    s.rolling(window=60).apply(mad, raw=True).plot(style='k')
 
+Numba as an argument
```

Comment:
Should the second title be something more like `Using the Numba engine`?  This sounds strange to me since Numba isn't an argument.

Reply:
I agree, will change.